README.tfdoc.hcl

```diff
@@ -147,7 +147,7 @@ section {
         }
 
         variable "source_repository" {
-          type        = object(source_repository)
+          type        = any
```

Comment:
please keep the object specification of terradoc, terradoc  will use this information to render the docs... this is not to confuse with terraform variable.type ;) happy to discuss details..